examples/python-typed/test_types.py

```
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+"""Test Python type stubs and mypy integration."""
+
+import sys
+sys.path.insert(0, '.')
+
+from typing import List, Optional
+from proto.gen.python import typed_pb2
+from proto.gen.python import typed_pb2_grpc
```

Ruff (0.11.9)

`proto.gen.python.typed_pb2_grpc` imported but unused

Remove unused import: `proto.gen.python.typed_pb2_grpc`

(F401)